dags/telemetry_aggregates.py

```diff
@@ -15,11 +15,26 @@
 
 dag = DAG('telemetry_aggregates', default_args=default_args, schedule_interval='@daily')
 
-telemetry_aggregate_view = EMRSparkOperator(
+prerelease_telemetry_aggregate_view = EMRSparkOperator(
     task_id = "telemetry_aggregate_view",
-    job_name = "Telemetry Aggregate View",
+    job_name = "Telemetry Aggregate View - Prerelease",
     instance_count = 10,
     execution_timeout=timedelta(hours=12),
-    env = {"date": "{{ ds_nodash }}"},
+    env = {
+      "date": "{{ ds_nodash }}",
+      "channels": "nightly, aurora, beta"
+    },
+    uri = "https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/run_telemetry_aggregator.sh",
+    dag = dag)
+
+release_telemetry_aggregate_view = EMRSparkOperator(
+    task_id = "telemetry_aggregate_view",
+    job_name = "Telemetry Aggregate View - Release",
+    instance_count = 40,
+    execution_timeout=timedelta(hours=12),
+    env = {
+      "date": "{{ ds_nodash }}",
+      "channels": "release"
```

Comment:
Should we include `esr` too?

Reply:
ESR isn't in the spec, and requires some changes to the aggregator. Let's confirm that's something people want and tackle that separately.

Reply:
Sounds good to me.